apps/asap-server/test/routes/news.route.test.ts

```diff
@@ -53,6 +53,13 @@ describe('/news/ route', () => {
     });
 
     describe('Parameter validation', () => {
+      test('Should return a validation error when additional fields exist', async () => {
+        const response = await supertest(app).get(`/news`).query({
```

Comment:
You can create new set of tests and put there all of the routers.
Then use test.each to validate invalid parameters.

Reply:
see other thread